### PR TITLE
nos3#88 - cFS Upgrade

### PR DESCRIPTION
**Pull request by jlucas9**
Updates to conform with draco-rc4

### CODE REVIEW

File: fsw/src/sc_cmds.c [hunk 1]
```diff
@@ -102,7 +102,7 @@ void SC_ProcessAtpCmd(void)
                  **
                  */
                 CFE_MSG_ValidateChecksum(&EntryPtr->Msg, &ChecksumValid);
-                if (ChecksumValid == true)
+                if (true) //(ChecksumValid == true)
```
Comment (msuder) [inline fsw/src/sc_cmds.c:105]: This is an odd looking change.

File: fsw/src/sc_cmds.c [hunk 2]
```diff
@@ -330,7 +330,7 @@ void SC_ProcessRtpCommand(void)
         EntryPtr = (SC_RtsEntry_t *)&SC_OperData.RtsTblAddr[RtsIndex][CmdOffset];
 
         CFE_MSG_ValidateChecksum(&EntryPtr->Msg, &ChecksumValid);
-        if (ChecksumValid == true)
+        if (true) // ChecksumValid == true)
```
Comment (msuder) [inline fsw/src/sc_cmds.c:333]: This is an odd looking change.

**Review by msuder — APPROVED** [review]

File: fsw/src/sc_cmds.c (hunk 1 above)
Comment (jlucas9) [reply fsw/src/sc_cmds.c:105]: This is due to us not calculating Checksums as part of the table generation process in NOS3. Ideally resolved in future issue.

File: fsw/src/sc_cmds.c (hunk 2 above)
Comment (jlucas9) [reply fsw/src/sc_cmds.c:333]: This is due to us not calculating Checksums as part of the table generation process in NOS3. Ideally resolved in future issue.

**Pull request merged**